pywwt_web/annotation.py

```diff
@@ -47,8 +47,57 @@ class Circle(Annotation):
 
     shape = 'circle'
 
-    def set_center(self, coord):
+    fill = Bool(False, help='Whether or not the circle should be filled').tag(wwt='fill', sync=True)
+    fill_color = Unicode('white', help='Assigns fill color for the circle').tag(wwt='fillColor', sync=True)
+    line_color = Unicode('white', help='Assigns line color for the circle').tag(wwt='lineColor', sync=True)
+    line_width = Float(1, help='Assigns line width (in pixels)').tag(wwt='linewidth', sync=True)
+    radius     = Float(10, help='Sets the radius for the circle').tag(wwt='radius', sync=True)
+    sky_relative = Bool(False, help='Whether the size of the circle is absolute or relative').tag(wwt='skyRelative', sync=True)
+
+    # def fill_color(self, color):
+    #     self.parent._send_msg(event='circle_set_fillColor', id=self.id,
+    #                           argb=color)
+
+    # def line_color(self, color):
+    #     self.parent._send_msg(event='circle_set_lineColor', id=self.id,
+    #                           argb=color) # correct?
+
+    # def line_width(self, width):
+    #     self.parent._send_msg(event='', id=self.id,
+    #                           lw=width)
```

Comment:
Could you remove these commented lines? (note that you can just push a new commit to this branch, you don't need to open a new pull request)